yaml/pois.yaml

```diff
@@ -1043,24 +1139,63 @@ filters:
     output:
       <<: *output_properties
       kind: gallery
+  - filter: { 'health_facility:type': health_centre }
+    min_zoom: 15
+    output:
+      <<: *output_properties
+      kind: health_centre
+      <<: *health_facility_type_kind_detail
+  - filter: { 'health_facility:type': dispensary }
```

Comment:
I wonder what this will do to medical marijuana "dispensary" in San Francisco and other regions that have legalized? Some map providers hide those from map display (or show with shopping icon) but allow in search.

Reply:
Although they're called "dispensaries", I can only find [one instance](https://www.openstreetmap.org/node/4942533433) where one has been mapped that way. The general consensus seems to be around [`shop=cannabis`](https://taginfo.openstreetmap.org/tags/shop=cannabis#overview), which is also a tag on that single instance.

Do we want to screen off / filter out `shop=cannabis`? Or give it a different kind so that it's easy for clients to turn them off in the stylesheet?

Reply:
If that's the case we're ignoring them for now and can decide to add them later. No action for now.